packages/picasso.js/src/core/chart-components/line/line.js

```diff
@@ -119,11 +93,15 @@ const SETTINGS = {
   },
 };
 
-function createDisplayLayer(points, { generator, item, data }, fill = '') {
-  const path = generator(points);
```

Comment:
basically, we want to delay generating path: moving the generating path logic from line nodes to paths (display object). This is because the path above is a string and D3 interpolation, which we used to generate transitioning nodes for animations, performs poorly on graphics presented as string. Instead, we should just pass points (objects) in here, let D3 interpolates the points, and only then generate the path from these points in packages/picasso.js/src/core/scene-graph/display-objects/path.js